xilinx/common/tcl/prologue.tcl

```diff
@@ -20,6 +20,9 @@ while {[llength $argv]} {
     -ip-vivado-tcls {
       set argv [lassign $argv[set argv {}] ip_vivado_tcls]
     }
+    -addl_constraint_dir {
```

Comment:
Let's match the same style as the other flags and use dashes instead of underscores.